source/HailPanel.cpp

```diff
@@ -55,25 +55,18 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship)
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	else if(gov->IsEnemy())
+	else if(gov->IsEnemy() && !ship->IsDisabled())
```

Comment:
Yes, it seems pretty unlikely to get a falldown, but if you are disabled/without fuel, and hail a hostile disabled ship, then you get this: